internal/linker/link_node_modules.bzl

```diff
@@ -162,7 +149,7 @@ def _get_module_mappings(target, ctx):
     # pkg_npm but not vice-verse at the moment since ts_library cannot handle directory artifacts as
     # deps.
     if hasattr(linkable_package_info, "_tslibrary") and linkable_package_info._tslibrary:
-        mr[0] = "_tslibrary"
+        mr[0] = "__tslibrary__"
```

Comment:
Not a change, but `mr[0]` reads in my head as "Mr. Zero" and it sounds like a Bond villain

Reply:
🔫